### ISSUE TITLE
Unscheduled Ctrl+c error

### ISSUE THREAD

**Issue by robbertvdg** [Theano/Theano#4907]
Hi,

I am running a network using my 970 GTX. It runs fine on the CPU, however crashes early on in the first epoch when using the GPU. The following error is shown:

    >>python net_baseline.py
    WARNING (theano.gof.compilelock): Overriding existing lock by dead process '1204' (I am process '11528')
    Using gpu device 0: GeForce GTX 970 (CNMeM is disabled, cuDNN 5005)
    Defining network
    Defining loss function
    Defining learning function
    Loading data
    training set size: data = (40000L, 3L, 32L, 32L), labels = (40000L,)
    validation set size: data = (10000L, 3L, 32L, 32L), labels = (10000L,)
    test set size: data = (10000L, 3L, 32L, 32L), labels = (10000L,)
    Determining mean and std of train set
    Training for 100 epochs
    28it [00:03,  9.38it/s]
    forrtl: error (200): program aborting due to control-C event
    Image              PC                Routine            Line        Source
    KERNELBASE.dll     00007FF8B1E86634  Unknown               Unknown  Unknown
    KERNEL32.DLL       00007FF8B3828102  Unknown               Unknown  Unknown
    ntdll.dll          00007FF8B69DC5B4  Unknown               Unknown  Unknown
    forrtl: error (200): program aborting due to control-C event
    Image              PC                Routine            Line        Source
    KERNELBASE.dll     00007FF8B1E86634  Unknown               Unknown  Unknown
    KERNEL32.DLL       00007FF8B3828102  Unknown               Unknown  Unknown
    ntdll.dll          00007FF8B69DC5B4  Unknown               Unknown  Unknown

Obviously I did not press Ctrl+c, is this a known problem and fixable?

Thanks in advance,

Robbert

**Comment by nouiz** [comment]
I would guess it is a problem with your environment, not in Theano.

Can you run other CUDA software? Can you run NVIDIA samples?

**Comment by robbertvdg** [comment]
I just tried the Mnist example from lasagne, which worked perfectly fine using the GPU. 

Can it be a code related issue? I find that strange since running on the cpu cause no problems.

**Comment by lamblin** [comment]
Just to be safe, can you make sure you do not have any older versions of the headers or libraries of cudnn on your system?
Does it happen if you disable cudnn completely by setting cudnn.enabled=False in the config flags?

**Comment by robbertvdg** [comment]
I re-installed theano and cudnn, however no improvement.. I can run the theano test file without troubles (http://deeplearning.net/software/theano_versions/dev/install_windows.html) but the GPU seems not much faster then CPU, which worries me. 

If i set cudnn.enabled=False in the .theanorc file it still shows cudnn enabled when importing theano.

**Comment by lamblin** [comment]
Again, just to be safe, you are using the correct syntax in the .theanorc file?
```
[cudnn]
enabled = False
```
What happens if you uninstall cudnn?

**Comment by robbertvdg** [comment]
Yes I added the lines to it, however it still says it is using cudnn. 

I got it working on linux, so it is not a coding problem, there has to be something wrong with my installation probably.

**Comment by nouiz** [comment]
ok. Closing as this isn't a problem in Theano.
